data.sql

```diff
@@ -27,3 +27,52 @@ INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg,sp
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg,species) VALUES (9, 'Boarmon', '2005-06-07', 7, 'true', 20.4, '' );
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg,species) VALUES (10, 'Blossom', '1998-10-13', 3, 'true', 17.0, '' );
 INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg,species) VALUES (11, 'Ditto', '2022-05-14', 4, 'true', 22.0, '' );
+
+
+-- Insert the following data into the owners table:
+insert into owners(full_name, age) values('Sam Smith', 34);
+insert into owners(full_name, age) values('Jennifer Orwell', 19);
+insert into owners(full_name, age) values('Bob', 45);
+insert into owners(full_name, age) values('Melody Pond', 77);
+insert into owners(full_name, age) values('Dean Winchester', 14);
+insert into owners(full_name, age) values('Jodie Whittaker', 38);
+
+-- Insert the following data into the species table:
+insert into species(name) values('Pokemon');
+insert into species(name) values('Digimon');
+
+/* Begin transaction */
+BEGIN;
+
+-- If the name ends in "mon" it will be Digimon
+update animals set species_id = 2 where name like '%mon';
+
+-- All other animals are Pokemon
+update animals set species_id= 1 where name not like '%mon';
+
+COMMIT;
+
+/* Begin transaction */
+BEGIN;
+
+-- Sam Smith owns Agumon.
+update animals set owner_id = 1 where name like 'Agumon';
+
+-- Jennifer Orwell owns Gabumon and Pikachu.
+update animals set owner_id = 2 where name like 'Gabumon';
+update animals set owner_id = 2 where name like 'Pikachu';
```

Comment:
- [ ] Optional: Instead of this you can use a single query using OR for upcoming projects.
```
UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';
```

Reply:
Thank you! I will look into it